.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
added this since the version 1.22 is erroring out 80% of the time with `expected xyz to exist in packages` when adding any package

Reply:
`yarn-path ".yarn/releases/yarn-1.19.0.cjs"`

I know this is autogenerated but won't this ever cause any conflict if someone has a different yarn-path?

Reply:
didn't try that 😞 
ideally is someone messes up with yarn path then logically this won't work and wouldn't fail as well as it'll pick yarn version from the globally set path

Reply:
Let's just use the engines key, otherwise, we'll end up doing this in all projects using yarn.
https://classic.yarnpkg.com/en/docs/package-json/#toc-engines

Reply:
There's a difference. The change here in `.yarnrc` will switch to this version by default when running this project vs the engines key will just make sure you have a particular version of yarn. We need people to switch to this exact yarn version while running this project